crowbar_framework/config/repos-cloud.yml

```diff
@@ -137,6 +117,26 @@ suse-12.1:
       url:
       smt_path: "SUSE/Updates/SLE-HA/12-SP1/x86_64/update"
       ask_on_error: false
+    suse-enterprise-storage-2.1-pool:
+      name: "SUSE-Enterprise-Storage-2.1-Pool"
+      required: "optional"
+      features: ["ceph"]
+      repomd:
+        tag: "obsproduct://build.suse.de/SUSE:SLE-12-SP1:Update:Products:SES2.1/ses/2.1/POOL/x86_64"
```

Comment:
If you go to the repositories page in the crowbar webui, it checks that the repo is valid. If it's all green, then you're good.